configs/timm_example/retinanet_timm-efficientnet-b1_fpn_1x_coco.py

```diff
@@ -4,9 +4,6 @@
     '../_base_/schedules/schedule_1x.py', '../_base_/default_runtime.py'
 ]
 
-# please install mmcls>=0.20.0
-# import mmcls.models to trigger register_module in mmcls
-custom_imports = dict(imports=['mmcls.models'], allow_failed_imports=False)
```

Comment:
Does it mean this PR can only be merged after MMCls support auto import?

Reply:
No

Reply:
No?

Reply:
Add some TODOs in the config. No need to wait for mmcls.